app/src/main/res/layout/winner_activity.xml

```diff
@@ -36,6 +36,15 @@
             android:text="Score: 11"
             android:textColor="@color/white"
             android:textSize="30sp"/>
+        <Button
+            android:id="@+id/startNew_Game_BTN"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/start_new_game"
```

Comment:
You can add a not all upper case option, look at the menu activity --> start button, 
look at the configuration of the button in the XML and you will find an option to let the string use case sensitive.